library/core-components/components/link-icon/link-icon.stories.tsx

```diff
@@ -27,6 +27,13 @@ const meta: Meta<typeof RadiusLinkIcon> = {
     className: { table: { disable: true } },
     ref: { table: { disable: true } },
   },
+  decorators: [
+    (Story) => (
+      <div style={{ display: 'flex' }}>
+        <Story />
+      </div>
+    ),
+  ],
```

Comment:
This is necessary to prevent the icon from extending the full width of the container (since it is a flex item itself)